### DISCUSSION TITLE
[FEEDBACK]: Integrate model skewing into data poisoning

### DISCUSSION THREAD

**Discussion by robvanderveer** [OWASP/www-project-machine-learning-security-top-10#109] [Machine Learning Security Topics]
### Type

Suggestions for Improvement

### What would you like to report?

[FEEDBACK]: Model skewing requires altering training data, making it a form of data poisoning. Therefore it is probably better to integrate the two threats.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by aryanxk02** [comment]
I agree with you @robvanderveer but then there will be only 9 security risks associated with the OWASP ML page. How do we merge the overlapping contents associated with both the attacks i.e Model Skewing and Data Poisoning?

**Reply by attilaswiss** [reply]
I absolutely agree with the merge. It was very hard to make a distinguish between the two.

**Comment by aryanxk02** [comment]
How about we merge these two attacks and introduce Excessive Agency @robvanderveer as mentioned in https://github.com/OWASP/www-project-machine-learning-security-top-10/issues/89?
